app/controllers/admin/topical_event_featurings_controller.rb

```diff
@@ -0,0 +1,100 @@
+class Admin::TopicalEventFeaturingsController < Admin::BaseController
+  before_action :load_topical_event
```

Comment:
So for organisations and world location news, they use 'Feature Lists' and a 'Feature Lists controller', by using `include Featureable`. So just wondering whether we could rationalise this even further by removing this altogether and using the same - I don't think there's a particular reason why classifications should have their own version of this, but maybe I'm missing something. But seems a bit fishy to me.

Reply:
Maybe we could do this as a separate PR to this but I think it'd be worth exploring to simplify things, as it's another bit of weirdness in Whitehall that we've encountered

Reply:
This is a really good idea for a spike or mob. Excellent spot